src/cppwinrt.build.pre.props

```diff
@@ -11,8 +11,8 @@
   <Import Project="$(OpenConsoleDir)\packages\Microsoft.Windows.CppWinRT.2.0.210309.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(OpenConsoleDir)\packages\Microsoft.Windows.CppWinRT.2.0.210309.3\build\native\Microsoft.Windows.CppWinRT.props')" />
 
   <PropertyGroup Label="Globals">
-    <!-- 17134 is RS4, 17763 is RS5, 18362 is 19H1 -->
-    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">10.0.18362.0</WindowsTargetPlatformVersion>
+    <!-- 18362 is 19H1, 19041 is 20H1 -->
+    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">10.0.19041.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion Condition="'$(WindowsTargetPlatformMinVersion)' == ''">10.0.18362.0</WindowsTargetPlatformMinVersion>

```

Comment:
Since we have converged the versions in cppwinrt.pre.props and common.pre.props we can probably removethe version declarations from this file. What do you think?

Reply:
I didn't notice it imports `common.build.pre.props`. 👍

Reply:
Ah lol nevermind... Since `cppwinrt.build.pre.props` specifies a `ApplicationTypeRevision` you cannot import the `WindowsTargetPlatformVersion` from somewhere else.